File: Programs-C/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Get size of pyramid
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    // Print grid of bricks
    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n; j++)
        {
            if (i + j < n - 1)
                printf(" ");
            else
                printf("#");
        }
        printf("\n");
    }
}

File: Programs Python/sentimental-mario-less/mario.py
# Get size of pyramid
while True:
    try:
        n = int(input("Height: "))
        if 1 <= n <= 8:
            break
    except ValueError:
        print("Enter a number 1-8")

# Print grid of bricks
for i in range(n):
    for j in range(n):
        if i + j < n - 1:
            print(" ", end="")
        else:
            print("#", end="")
    print()
